sdk/dotnet/Notebooks/Instance.cs

```diff
@@ -336,15 +336,21 @@ public sealed class InstanceArgs : Pulumi.ResourceArgs
         [Input("installGpuDriver")]
         public Input<bool>? InstallGpuDriver { get; set; }
 
+        [Input("instanceOwners")]
+        private InputList<string>? _instanceOwners;
+
         /// <summary>
-        /// The owner of this instance after creation.
+        /// The list of owners of this instance after creation.
         /// Format: alias@example.com.
         /// Currently supports one owner only.
         /// If not specified, all of the service account users of
         /// your VM instance's service account can use the instance.
         /// </summary>
-        [Input("instanceOwners")]
-        public Input<string>? InstanceOwners { get; set; }
+        public InputList<string> InstanceOwners
```

Comment:
That's a breaking change here

Reply:
Oh, we can mitigate this with MaxItems: 1, the comment even says it only supports one owner

I will fix this up now